Shaders/qUINT_bloom.fx

```diff
@@ -21,7 +21,7 @@
 =============================================================================*/
 
 #ifndef SAMPLE_HQ
- #define SAMPLE_HQ 1
```

Comment:
This is intentionally set to 1 in GShade, as the primary concern of the build is Final Fantasy XIV support.